ait/gui/static/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "ait-gui",
-  "version": "2.3.1",
+  "version": "2.3.2-rc1",
```

Comment:
do we care about this version at all? should it match the version we set in `setup.py`? I think this file is actually overwritten by https://github.jpl.nasa.gov/SunRISE-Ops/SunRISE-AIT/blob/master/gui/static/package.json in SunRISE-AIT. But, question still remains for this repo